web/src/security/LoginForm.js

```diff
@@ -99,7 +98,7 @@ class LoginForm extends Component {
             api: { isExecuting: true },
             
         }, () => {
-            axios.post(getEnvironment().apiRoot + '/v1/security/login', this.state)
+            api.post('/v1/security/login', this.state)
```

Comment:
Undocumented and subtle note here, using `api` will cause a login failure to be attempted twice.  I'm not sure its a big concern that it does so.

If you end up creating a wrapper over axios you can allow a parameter that suppresses retries.